CorsixTH/Lua/humanoid_actions/meander.lua

```diff
@@ -54,6 +54,12 @@ local function meander_action_start(action, humanoid)
   if humanoid.humanoid_class == "Doctor" or humanoid.humanoid_class == "Nurse" then
     if not room then
       humanoid:setDynamicInfoText(_S.dynamic_info.staff.actions.wandering)
+    -- Answering call queue when in a room:
+    elseif room.room_info.id ~= "staff_room" and humanoid:isIdle() and not room:getPatient() and room.door.queue:patientSize() == 0 and humanoid.world.dispatcher:answerCall(humanoid) then
```

Comment:
This line is very long, please split it according to the coding convention (after <code>and</code>)